keyboards/naked48/keymaps/default/rules.mk

```diff
@@ -0,0 +1 @@
+SPLIT_KEYBOARD = no
```

Comment:
?

Reply:
For keymaps that are not using the split keyboard feature, the corresponding feature is turned off.
This keyboard can add a numeric keypad using the split keyboard feature.